admin/src/components/connect-banner.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from '@atlaskit/image';
+import { css } from '@emotion/react';
+
+export function ConnectBanner() {
+	return (
+		<div
+			css={css`
+				display: flex;
+				align-items: center;
+			`}
+		>
+			<div css={css({ padding: 16 })}>
```

Comment:
I should probably know this but does a raw number like `16` work with emotion's `css` prop or do you need to specify the unit of measurement as well?

Reply:
the raw number works here! but only if we're passing an object to `css()`. If we're using tagged template literals, we need to pass in px since the string needs to be valid CSS